ModuleInspector.hs

```diff
@@ -156,7 +155,10 @@ defInfo (H.PatBind loc pat _ _ _) = map (\name -> DeclarationInfo loc "function"
 
     fieldNames :: H.PatField -> [H.Name]
     fieldNames (H.PFieldPat _ n) = names n
-    fieldNames (H.PFieldPun n) = [n]
+    fieldNames (H.PFieldPun n0) 
+      | H.Qual _ n <- n0 = [n]
```

Comment:
This is the line I am unsure on.  The wildcard is the `ModuleName` part of the qualified field name, but I could find no reasonable way to incorporate that into the existing SublimeHaskell api.  It may make more sense to base everything else on `QName`s rather than `Name`s, but that would be quite an extensive change.